test/functional/feature_assumeutxo.py

```diff
@@ -350,6 +350,29 @@ def run_test(self):
         assert_equal(loaded['coins_loaded'], SNAPSHOT_BASE_HEIGHT)
         assert_equal(loaded['base_height'], SNAPSHOT_BASE_HEIGHT)
 
+        self.log.info("Check that UTXO-querying RPCs operate on snapshot chainstate")
+        # coinstatsindex might be not caught up yet and is not relevant for this test, so don't use it
+        utxo_info = n1.gettxoutsetinfo(use_index=False)
+        assert_equal(utxo_info['txouts'], loaded['coins_loaded'])
+        assert_equal(utxo_info['height'], loaded['base_height'])
+        assert_equal(utxo_info['bestblock'], loaded['tip_hash'])
+
+        # find coinbase output at snapshot height on node0 and scan for it on node1,
+        # where the block is not available, but the snapshot was loaded successfully
+        snapshot_hash = loaded['tip_hash']
```

Comment:
nit: could have moved this up and used it in the `gettxoutsetinfo` as well

Reply:
On second thought: maybe just pull all three of these loaded keys out in the beginning since you use each of them at least twice and they are used again further below... or you could deduplicate the `assert_equal` triplet but it's not a big deal either way...